build.gradle

```diff
@@ -138,3 +138,11 @@ slackSendMessage {
         text: messageText
     )
 }
+
+task configure(type: Exec) {
+    File attributeFile = new File("${projectDir}/.git/info/attributes")
+    attributeFile.write("*.groovy filter=tabspace\n")
+    commandLine "git", "config", "filter.tabspace.clean", "unexpand -t 2"
```

Comment:
I know we still have code that uses spaces, does this affect code outside of the users changes? (I'm not familiar with this git config.)

Reply:
It only works on files included in the commit.